### PR TITLE
Various /alerts updates

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205516319506317
  - https://app.asana.com/0/0/1205535207353136

### CODE REVIEW

File: docs/alerts/monitors/overview.md [hunk 1]
```diff
@@ -11,23 +11,45 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
-* Metric Monitors delay execution by one minute.
-* Depending on your account type, you can have up to a certain number of Log and Metric Monitors.
-  * Enterprise and Trial can have up to 1,000 Log Monitors and 1,500 Metric Monitors.
-  * Essentials and Professional can have up to 300 Log Monitors and 500 Metric Monitors.
-  * Free can have up to 50 Log Monitors and 50 Metric Monitors.
+* Enterprise and Trial plan customers can have up to 1,000 Log Monitors.
+* Essentials and Professional plan customers can have up to 300 Log Monitors.
+* Free Trial customers can have up to 50 Log Monitors.
+
+#### Auto-Resolving Notifications
```
Comment (jpipkin1) [inline docs/alerts/monitors/overview.md:38]:
```suggestion
#### Auto-resolving notifications
```

**Review by jpipkin1 — APPROVED** [review]
Looks good, @kimsauce. I have a tiny suggestion.

File: docs/alerts/monitors/monitor-faq.md [hunk 2]
```diff
@@ -73,12 +73,3 @@ One reason could be that the user who created the monitor was deleted. You can c
 You can quickly **Duplicate** the monitor by hovering over it on the Monitors page and clicking the three-dot icon:<br/>![more actions menu for monitors.png](/img/monitors/more-actions-menu-for-monitors.png)  
 
 then selecting **Duplicate**. If your monitor still doesn't work, we recommend contacting [Sumo Logic support](https://support.sumologic.com/). 
-
-## Can I disable a Monitor during scheduled maintenance or upgrade window?
```
Comment (tarunk2) [inline docs/alerts/monitors/monitor-faq.md:77]: Should we retain this question and point the answer to scheduled muting docs for it?

Comment (kimsauce) [reply docs/alerts/monitors/monitor-faq.md:77]: Great thinking, Tarun! Done.

**Pull request merged**